### DISCUSSION TITLE
Activate DND programmatically

### DISCUSSION THREAD

**Discussion by javar0ne** [zepp-health/zeppos-forum#278] [Q&A] [answered]
Hi guys,
just wondering if it is possible to activate DND mode programmatically. I don't see any example in the docs and cannot find any on github.
If it is possible, could you please provide me some examples? thanks

**Comment by silver-zepp** [comment]
Not possible and no plans to allow this as it might introduce confusion to our user base when 3rd party apps start enabling/disabling DND @javar0ne

**Reply by javar0ne** [reply]
Oh I see, I use timed DND for work but cannot set days for which it should not be enabled (i.e. in the weekend). Any chance it will be possible in the future?

**Reply by silver-zepp** [reply, marked as answer]
I see. I'll forward your suggestion to our App team. Keep in mind there will be no ETA on the resolution of this case even if the feature gets accepted.
